examples/README.md

```diff
@@ -40,6 +40,7 @@ information on functionality and ordering.
 | [Handlers](handlers) | Developing application logic that responds to web requests. | 4 |
 | [Middleware](middleware) | Developing custom middleware for your application. | 1 |
 | [Into Response](into_response) | Implementing the Gotham web framework's `IntoResponse` trait. | 1 |
+| [Tera Template](tera_template) | An example using Tera template engine | 1 |
```

Comment:
I think this might warrant a new `templating` directory for examples, so that we can add examples for other engines as we need to. Do you want to add this directory (you can base it on one of the other directories). Then your example can just be named `tera`.